src/include/network/connection_dispatcher_task.h

```diff
@@ -37,7 +38,8 @@ class ConnectionDispatcherTask : public common::NotifiableTask {
    * @param thread_registry DedicatedThreadRegistry dependency needed because it eventually spawns more threads in
    * RunTask
    */
-  ConnectionDispatcherTask(uint32_t num_handlers, int listen_fd, common::DedicatedThreadOwner *dedicated_thread_owner,
+  ConnectionDispatcherTask(bool hot_standby, uint32_t num_handlers, int listen_fd,
```

Comment:
Why is this flag needed for the connection dispatcher? What other protocols need this?

Reply:
The flag will tell the dispatcher whether to listen for replication messages (start event loop on replication port)

Reply:
@tpan496 Lets remove the flag for now.